src/ip/Intercept.cc

```diff
@@ -204,7 +204,7 @@ Ip::Intercept::IpfInterception(const Comm::ConnectionPointer &newConn, int silen
     memset(&natLookup, 0, sizeof(natLookup));
     // for NAT lookup set local and remote IP:port's
     if (newConn->remote.isIPv6()) {
-#if IPFILTER_VERSION < 5000003
+#if HAVE_NATLOOKUP_NL_INIPADDR_IN6
```

Comment:
I know nothing about these macros, but please double check this condition -- it _looks_ like it is reversed:

```
#if HAVE_FOO
    complain that we do not have something we need
#else
    // OK, so we do not have FOO, but we use foo in an assignment?!
    newConn->local.getInAddr(natLookup.nl_inipaddr.in6);
#endif
```